pkg/dataplane/test/sync_test.go

```diff
@@ -1137,6 +1139,49 @@ func (suite *syncStreamTestSuite) TestStream() {
 
 	response.Release()
 
+	getItemsInput := v3io.GetItemsInput{
+		Path:               streamPath,
+		ReturnData:         "true",
+		AllowObjectScatter: "true",
+		AttributeNames:     []string{"**"},
+	}
+
+	suite.populateDataPlaneInput(&getItemsInput.DataPlaneInput)
+
+	cursor, err := v3io.NewItemsCursor(suite.container, &getItemsInput)
+	suite.Require().NoError(err, "Failed to get items")
+
+	cursorItems, err := cursor.AllSync()
+	suite.Require().NoError(err)
+
+	// internal use case test
+	for _, cursorItem := range cursorItems {
+		shardName, err := cursorItem.GetFieldString("__name")
+		suite.Require().NoError(err, "Failed to get item name")
+		chunkId, streamData, chunkMetadata, currentChunkMetadata, err := cursorItem.GetShard()
```

Comment:
how will the user of GetShard add to streamData (from several responses..)

Reply:
the user need to append the all the data for the same chunk_id from several Items.